src/Pos/Block/Network/Listeners.hs

```diff
@@ -78,8 +78,8 @@ handleBlockHeaders
     :: forall ssc m.
        (SscWorkersClass ssc, WorkMode ssc m)
     => (ListenerSpec m, OutSpecs)
-handleBlockHeaders = listenerConv $ \__ourVerInfo nodeId conv -> do
+handleBlockHeaders = listenerConv @Void $ \__ourVerInfo nodeId conv -> do
```

Comment:
This change is dangerous. You should ensure other side of conversation also expects Void